package/yast2-crowbar.changes

```diff
@@ -72,7 +78,7 @@ Wed Oct  7 13:52:52 CEST 2015 - jsuchome@suse.com
 
 - Adapt module so it can be called during first stage of installation
   (which is the only stage in SLES12)
-- 3.1.8 
+- 3.1.8
```

Comment:
this small changes are fine, it is usually ignored, also typos, but you should not rephrase it nor change it too much